### DISCUSSION TITLE
Training Log 2023-01-15

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#308] [Training Logs 训练日志]
# CPM-Live Training Log (January, 15)

Time: January, 15 2023 19:00

Recorder: @zh-zheng

## Loss
- Begin: 2.07
- End: 2.06
	
## Completed Data
- $\approx$ 2217.08GB

## Average Grad Norm
- 1.20

## Progress
- 92.36%

## Comment

I sometimes wonder if CPM-Bee's fill-in-the-blank ability has surpassed mine.

**Comment by fengyunzaidushi** [comment]
great!
